content/docs/intro/vs/cloud_template_transpilers.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Pulumi vs. Cloud Template Transpilers (AWS CDK, Troposphere)
+linktitle: Cloud Template Transpilers
+menu:
+  intro:
+    parent: vs
+    weight: 3
+---
+
+Because of [the challenges of writing raw YAML/JSON by hand]({{< relref "cloud_templates.md" >}}), two notable
+projects exist to compile higher level languages into AWS CloudFormation YAML/JSON templates:
+
+* **Troposphere**: a community-led open source project created in 2013
+* **AWS Cloud Development Kit (CDK)**: an AWS Labs project created in 2018
+
+Similar to Pulumi, these projects let you author infrastructure as code using general purpose languages like TypeScript,
+JavaScript, and Python. Unlike Pulumi, however, whose open source engine understands these languages, a _transpiler_
+(a.k.a., [_source-to-source compiler_](https://en.wikipedia.org/wiki/Source-to-source_compiler)) translates this program
+into [AWS CloudFormation YAML/JSON]({{< relref "cloud_templates.md" >}}). This resulting markup file is then submitted
+to the closed source AWS CloudFormation servers to provision infrastructure on AWS in the usual ways.
+
+## Summary of Differences
+
+The transpiler approach gives you some of the benefits of Pulumi, with the following caveats:
+
+* Troposphere/CDK only support the AWS platform. Pulumi supports many clouds, including: major cloud platforms
+  such as Microsoft Azure, Google Cloud Platform, Kubernetes, and DigitalOcean; on-premises and hybrid technologies
+  such as VMWare vSphere and OpenStack; and online SaaS offerings like Cloudflare, Datadog, New Relic, and more.
+  Furthermore, Pulumi is extensible, supports custom providers, and can bridge with any existing Terraform-based provider.
+
+* Pulumi supports Cloud Native technologies, including Kubernetes, Helm Charts, Istio service
+  meshes, and hosted Kubernetes clusters in any cloud (AWS EKS, Azure AKS, Google GKE, etc).
+
+* Troposphere/CDK compiles down to YAML and is therefore limited in what it can express. The Pulumi engine understands
+  general purpose language patterns, dependencies between objects, and therefore delivers a better overall experience.
+  Pulumi also supports going beyond what you can express in YAML, such as building/publishing Docker container image,
+  authoring serverless functions in code and automating packaging and versioning of code, and so on.
+
+* Pulumi's engine is [open source](https://github.com/pulumi/pulumi), whereas Troposphere/CDK depend on the closed
+  source CloudFormation engine. This means more of Pulumi is accessible to community contributions.
+
+* The Pulumi CLI and Console are co-designed to [make team collaboration simple]({{< relref "/docs/intro/console" >}}),
```

Comment:
Can we highlight some substantial benefits of the Pulumi CLI?

Reply:
Happy to! What are those in your mind?

Reply:
I don't have much experience with CloudFormation to do so. I guess you already covered some in other bullet points.